### DISCUSSION TITLE
cpptool very slow

### DISCUSSION THREAD

**Discussion by piterzhang** [microsoft/vscode-cpptools#10435] [Q&A]
1.when I use cpptool to read linux code,the speed of go to function define is very slowly and some times can't work
2. then i try gun global in vscode to read linux code ， surprised ,very fast. but used gun global cant used cpptool,I need cpptool to debug also

Q:

1.can give user a selectc used cpptool or gun global

2.if cpptool can work like gun global, generate data base before read code
let it fast when read large project

**Comment by sean-mcmanus** [comment]
This is a duplicate of https://github.com/microsoft/vscode-cpptools/discussions/10397 .
